app/src/main/assets/html/private.html

```diff
@@ -0,0 +1,31 @@
+<html>
+    <head>
+        <title>Private Browsing</title>
+        <link rel="stylesheet" href="css/styles.css"/>
+    </head>
+    <body data-page="private">
+        <div class="container">
+            <img class="icon" src="images/private-browsing.svg" />
+
+            <h1>Private Browsing</h1>
+
+            <p>When you browse in a Private Window. Firefox Reality <strong>does not save</strong>:</p>
+
+            <ul class="two-column">
+                <li>visited pages</li>
+                <li>cookies</li>
+                <li>searches</li>
+                <li>temporary files</li>
+            </ul>
+
+            <p>Firefox Reality <strong>will save</strong> your:</p>
```

Comment:
Don't currently have support for downloads and bookmarks. We should remove that part until we get that.

Reply:
OK cool.  Will do that.